test/integration/lerna-publish.test.js

```diff
@@ -57,4 +60,15 @@ describe("lerna publish", () => {
       });
     });
   }));
+
+  test("updates independent versions by npm", () => {
+    return initFixture("PublishCommand/integration").then((cwd) => {
+      return Promise.resolve()
+        .then(() => installInDir(cwd))
+        .then(() => execa("npm", ["run", "lp"], { cwd }))
```

Comment:
@evocateur Could this (and the other new integration tests) be considered as a "proof" that the new Command and arg mechanics don't break existing functionality?